ui/app/components/send_/send.utils.js

```diff
@@ -45,9 +45,9 @@ function calcGasTotal (gasLimit, gasPrice) {
 
 function isBalanceSufficient ({
   amount = '0x0',
-  amountConversionRate = 0,
```

Comment:
I wonder why `amountConversionRate` was defaulted to `0` before? We should confirm that changing the default to `1` does not break any desired behaviour.